index.html

```diff
@@ -6,6 +6,7 @@
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/style.css">
   <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="vendor/nouislider/nouislider.css">
   <title>Кекстаграм</title>
```

Comment:
адреса относительные указывай 
```suggestion
  <link rel="stylesheet" href="./vendor/nouislider/nouislider.css">
  <title>Кекстаграм</title>
```